ext/azuredevops/setupAzd/tests/InstallSuccess.ts

```diff
@@ -1,7 +1,7 @@
 import * as path from 'path';
 import * as assert from 'assert';
 import * as ttm from 'azure-pipelines-task-lib/mock-test';
-import * as fs from 'fs'
+import * as fs from 'fs';
 
 describe('setup azd tests', function () {
     setTimeout(() => { }, 10000);
```

Comment:
Is this supposed to control the test timeout? If so we may need to use `this.timeout(...)`: https://mochajs.org/#timeouts

Reply:
Usually `setTimeout` is used to delay the execution of the function (first argument), but here it's empty: `() => { }` which I don't quite understand - in the documentation I think `setTimeout` is used to simulate a function that takes `x` milliseconds to complete

Reply:
Glad that worked!